Assignment/solution_student/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,57 @@
 // 1.	Slide the header up when it is clicked on.
-
+$('h1').on('click', function () {
```

Comment:
No big deal, because you got the point down, but we were looking for you to target the `<header>` here :)